### DISCUSSION TITLE
Deco 01 V2 wrong tablet area

### DISCUSSION THREAD

**Discussion by AlexC0903** [OpenTabletDriver/OpenTabletDriver#1571] [Q&A] [answered]
I am aware there is a another thread like this that exists, but it is unanswered so I am making one more. I normally use a sens of 55x30.94, in hawku driver, but since it's pretty outdated, I swited to OTD, but for some reason 55x30.94 is double the sens of 55x30.94 in hawku driver. When I use full area, the full area of the driver treats it like I'm using a quarter area, only moving my cursor in 1/4 of my screen. I have a two monitor setup, both 1920x1080, and I have set my affected monitor to only 1.

![image](https://user-images.githubusercontent.com/44229612/139533473-2e284021-8e1f-4068-a2c2-ff853a55859d.png)

**Comment by jamesbt365** [comment, marked as answer]
As fair as i know, this problem is solved in `master`.
Extract this and try putting this config in your `configurations\xp-pen` located where otd is located.
[Deco.01.V2.zip](https://github.com/OpenTabletDriver/OpenTabletDriver/files/7447439/Deco.01.V2.zip)

If this doesn't fix it export a diag at the top of otd by going to help > export diagnosis. Save that file as a txt and send it here.

This could also be another driver installed.

**/edit by gonX:** The tablet should be officially supported with OpenTabletDriver v0.6.1 and newer.
